src/main/java/step3/Racing.java

```diff
@@ -0,0 +1,46 @@
+package step3;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Racing {
+    private final int carNumber;
+    private final int tryNumber;
+    private final CarList carList;
+    private static final RandomCreator randomCreator = new RandomCreator();
```

Comment:
상수 이므로 대문자 선언되면 더 좋을 것 같습니다 🙏 
```suggestion
    private static final RandomCreator RANDOM_CREATOR = new RandomCreator();
```